Userland/Libraries/LibC/mallocdefs.h

```diff
@@ -77,3 +77,6 @@ struct ChunkedBlock : public CommonHeader {
 
     using List = IntrusiveList<&ChunkedBlock::m_list_node>;
 };
+
+// FIXME: Clang complains about not using this.
+__attribute__((used)) static thread_local bool s_allocation_enabled = true;
```

Comment:
Variables should live in a translation unit, not in the header, as everything that includes `mallocdefs.h` would then get its own copy.

Please see how `errno` is defined.

Also, `static` means that this won't be seen outside the translation unit it's defined in. That's almost definitely not what you want to do, and that's why Clang complains. 

The `__attribute__((used))` will be rendered unnecessary if you address this problem.

Reply:
I'm not used to global variables, which I guess is a good thing (except here)...